### ISSUE TITLE
[Bug]: Stuck restarting every 7 seconds "Main process exited, code=killed, status=7/BUS"

### ISSUE THREAD

**Issue by Aadniz** [netdata/netdata#19061]
### Bug description

There is a certain VPS provider where I have experienced netdata suddenly becoming into an unusable state after some weeks of use, restarting indefinitely. The issue will occur after a restart

Looking at `journalctl -fu netdata.service`, there is this output that repeats every 7 seconds:
```
Nov 22 10:20:37 kusari-ab2289 systemd[1]: netdata.service: Scheduled restart job, restart counter is at 1669.
Nov 22 10:20:37 kusari-ab2289 systemd[1]: Stopped netdata.service - netdata - Real-time performance monitoring.
Nov 22 10:20:37 kusari-ab2289 systemd[1]: netdata.service: Consumed 2.349s CPU time.
Nov 22 10:20:37 kusari-ab2289 systemd[1]: Started netdata.service - netdata - Real-time performance monitoring.
Nov 22 10:20:37 kusari-ab2289 netdata[606813]: CONFIG: cannot load cloud config '/var/lib/netdata/cloud.d/cloud.conf'. Running with internal defaults.
Nov 22 10:20:37 kusari-ab2289 netdata[606813]: 2024-11-22 10:20:37: netdata INFO  : MAIN : CONFIG: cannot load cloud config '/var/lib/netdata/cloud.d/cloud.conf'. Running with internal defaults.
Nov 22 10:20:43 kusari-ab2289 systemd[1]: netdata.service: Main process exited, code=killed, status=7/BUS
Nov 22 10:20:43 kusari-ab2289 systemd[1]: netdata.service: Killing process 606815 (netdata) with signal SIGKILL.
Nov 22 10:20:43 kusari-ab2289 systemd[1]: netdata.service: Killing process 606991 (bash) with signal SIGKILL.
Nov 22 10:20:43 kusari-ab2289 systemd[1]: netdata.service: Killing process 607015 (nfacct.plugin) with signal SIGKILL.
Nov 22 10:20:43 kusari-ab2289 systemd[1]: netdata.service: Killing process 607016 (apps.plugin) with signal SIGKILL.
Nov 22 10:20:43 kusari-ab2289 systemd[1]: netdata.service: Killing process 607030 (python.d.plugin) with signal SIGKILL.
Nov 22 10:20:43 kusari-ab2289 systemd[1]: netdata.service: Killing process 606819 (netdata) with signal SIGKILL.
Nov 22 10:20:43 kusari-ab2289 systemd[1]: netdata.service: Failed with result 'signal'.
Nov 22 10:20:43 kusari-ab2289 systemd[1]: netdata.service: Consumed 3.199s CPU time.
```

The issue occurred this morning after the VPS was restarted by an administrator of the system.

### Expected behavior

Netdata should not be stuck in an infinite restarting state.

### Steps to reproduce

Not entirely sure how to reproduce, other than using this VPS's specific setup. Netdata has always worked fine in all other cases (other VPS services, other physical servers)

1. Use the setup of the VPS vm
2. `apt install netdata`
3. Wait a month or so
4. Restart (presumably with an unexpected/unclean shutdown)

### Installation method

from source

### System info

```shell
Linux kusari-ab2289 6.1.0-27-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.1.115-1 (2024-11-01) x86_64 GNU/Linux
/etc/cloud-release:ID=generic
/etc/cloud-release:VERSION="20241015-1901"
/etc/os-release:PRETTY_NAME="Debian GNU/Linux 12 (bookworm)"
/etc/os-release:NAME="Debian GNU/Linux"
/etc/os-release:VERSION_ID="12"
/etc/os-release:VERSION="12 (bookworm)"
/etc/os-release:VERSION_CODENAME=bookworm
/etc/os-release:ID=debian
```

### Netdata build info

```shell
Version: netdata v1.37.1
Configure options:  '--build=x86_64-linux-gnu' '--prefix=/usr' '--includedir=${prefix}/include' '--mandir=${prefix}/share/man' '--infodir=${prefix}/share/info' '--sysconfdir=/etc' '--localstatedir=/var' '--disable-option-checking' '--disable-silent-rules' '--libdir=${prefix}/lib/x86_64-linux-gnu' '--runstatedir=/run' '--disable-maintainer-mode' '--disable-dependency-tracking' '--libdir=/usr/lib' '--libexecdir=/usr/lib' '--disable-cloud' '--enable-x86-sse' '--disable-exporting-prometheus-remote-write' '--enable-dbengine' '--enable-https' '--enable-plugin-cups' '--enable-plugin-nfacct' '--with-math' 'build_alias=x86_64-linux-gnu' 'CFLAGS=-g -O2 -ffile-prefix-map=/build/netdata-aGlU9O/netdata-1.37.1=. -fstack-protector-strong -Wformat -Werror=format-security -Wall -O3' 'LDFLAGS=-Wl,-z,relro -Wl,-z,now -Wl,--as-needed' 'CPPFLAGS=-Wdate-time -D_FORTIFY_SOURCE=2' 'CXXFLAGS=-g -O2 -ffile-prefix-map=/build/netdata-aGlU9O/netdata-1.37.1=. -fstack-protector-strong -Wformat -Werror=format-security'
Install type: unknown
Features:
    dbengine:                   YES
    Native HTTPS:               YES
    Netdata Cloud:              NO (by user request)
    ACLK:                       NO
    TLS Host Verification:      YES
    Machine Learning:           NO
    Stream Compression:         YES
Libraries:
    protobuf:                YES (system)
    jemalloc:                NO
    JSON-C:                  YES
    libcap:                  YES
    libcrypto:               YES
    libm:                    YES
    tcalloc:                 NO
    zlib:                    YES
Plugins:
    apps:                    YES
    cgroup Network Tracking: YES
    CUPS:                    YES
    EBPF:                    NO
    IPMI:                    NO
    NFACCT:                  YES
    perf:                    YES
    slabinfo:                YES
    Xen:                     NO
    Xen VBD Error Tracking:  NO
Exporters:
    AWS Kinesis:             NO
    GCP PubSub:              NO
    MongoDB:                 NO
    Prometheus Remote Write: NO
Debug/Developer Features:
    Trace Allocations:       NO
```

### Additional info

Output of `tail -F /var/log/netdata/error.log` for 1 minute: https://gist.github.com/Aadniz/f063c25f39d53440f234455949b66add

Config is untouched (default provided by Debian)

**Comment by ilyam8** [comment]
@Aadniz hey. We don't support Netdata v1.37.1. Consider upgrading to the latest stable version.

**Comment by Aadniz** [comment]
Thanks, tested with kickstarter.sh instead of apt and the problem is fixed (version v2.0.0-91-nightly)
